src/compiler/types.ts

```diff
@@ -5744,7 +5744,6 @@ export const enum SymbolFlags {
     Transient               = 1 << 25,  // Transient symbol (created during type check)
     Assignment              = 1 << 26,  // Assignment treated as declaration (eg `this.prop = 1`)
     ModuleExports           = 1 << 27,  // Symbol for CommonJS `module` of `module.exports`
-    /** @internal */
     All = FunctionScopedVariable | BlockScopedVariable | Property | EnumMember | Function | Class | Interface | ConstEnum | RegularEnum | ValueModule | NamespaceModule | TypeLiteral

```

Comment:
I wonder why `All` isn't just `-1` 🤔

Reply:
We noticed that because this isn't `-1`, we're missing `Assignment` and `ModuleExports`. But, adding those does not appear to matter for our tests and are probably an oversight.

I just tried with `-1` and that also worked.

Reply:
IMO let’s change it to `-1` and then it won’t be out of date if we add any more flags.